Block/RecentPostsBlockService.php

```diff
@@ -34,6 +34,11 @@ class RecentPostsBlockService extends BaseBlockService
     protected $manager;
 
     /**
+     * @var Pool
+     */
+    private $adminPool;
```

Comment:
Which commit do you mean? @jordisala1991 

I think there is no problem if we have a private property and someone else is declarring the same property as public.